client/www/scripts/modules/profiler/profiler.services.js

```diff
@@ -94,7 +94,9 @@ Profiler.service('ProfilerService', [
 
             if ( data.status === 200 ) {
               $interval.cancel(intv);
-              var file = new File([data.data], fileName, { type: "text/plain" });
+              //var file = new File([data.data], fileName, { type: "text/plain" });
```

Comment:
Can you take this line out? We can pull it out of the repo history if we need to restore it.